terms.tsv

```diff
@@ -361,3 +361,5 @@ CONSO00359	Sandra	TDP-43 aggregates	cellular component	pmid:30824544	TDP-43 aggr
 CONSO00360	Yojana	Clonal Expansion	pathology	pmid:27794524	Multiplication or reproduction by cell division of a population of identical cells descended from a single progenitor. In immunology, may refer to the clonal proliferation of cells responsive to a specific antigen as part of an immune response.
 CONSO00361	Charlie	CDR - Sum of Boxes	clinical measurement	pmid:25468261, pmid:23626081, pmid:27084223	Clinical Dementia Rating (CDR) Sum of Boxes
 CONSO00362	Yojana	Tau antibody, MC1	antibody	pmid:28148553, pmid:23638068, pmid:21841002	mouse monoclonal antibody specific to tau epitopes within 312 to 322 amino acid
+CONSO00363	Yojana	RGFP136	chemical	pmid:23356410, pmid:21228185	HDAC3 inhibitor
+CONSO00364	Yojana	HDAC-3 inhibitor	chemical	pmid:23356410, pmid:21228185	Compounds that inhibit Histone Deactelylase 3 (HDAC-3) 
```

Comment:
Switch type to [chemical role](https://github.com/pharmacome/conso/blob/master/classes.tsv#L12)